src/test/resources/application.properties

```diff
@@ -3,3 +3,6 @@ spring.datasource.driverClassName=org.h2.Driver
 spring.datasource.username=sa
 spring.datasource.password=password
 spring.jpa.database-platform=org.hibernate.dialect.H2Dialect
+
+jwt.expiration=100000
+jwt.secret=qwerty1234qwerty1234qwerty1234qwerty1234qwerty1234qwerty1234
```

Comment:
It's better to get sensitive info from system properties or some other safe place instead of pushing it to git.

Reply:
Yeah but, then ci will not pass

Reply:
It's main application.properties. For CI test application.properties are used where you can put any mock secrets and it's OK.